appveyor.yml

```diff
@@ -38,10 +38,11 @@ build_script:
 test_script:
   - yarn test:setup
   - ./script/test-appveyor.bat
-  - yarn test:report
 
 on_success:
+  - yarn test:report
```

Comment:
i don't think we need this line, since `on_finish` is [always called](https://www.appveyor.com/docs/build-configuration/#build-pipeline) (on success and failure)

Reply:
Good spot! Fixed in efdfb2891.